### PR TITLE
ISPN-11766 Fix when the server is down

### PR DESCRIPTION
**Pull request by karesti**
Run the app without any server and check the console

https://issues.redhat.com/browse/ISPN-11766

### CODE REVIEW

File: src/app/GlobalStats/GlobalStats.tsx [hunk 1]
```diff
@@ -99,24 +100,28 @@ const GlobalStats: React.FunctionComponent<any> = props => {
...
-            Statistics are not enabled. To activate statistics, set
-            statistics=true in the configuration.
-          </EmptyStateBody>
-        </EmptyState>
+        <Card>
+          <CardBody>
+            <EmptyState variant={EmptyStateVariant.full}>
+              <EmptyStateIcon icon={CubesIcon} />
+              <Title headingLevel="h5" size="lg">
+                Statistics are not enabled
```
Comment (pruivo) [inline src/app/GlobalStats/GlobalStats.tsx:116]: suggestion: `Statistic disabled`

File: src/app/GlobalStats/GlobalStats.tsx [hunk 2]
```diff
@@ -99,24 +100,28 @@ const GlobalStats: React.FunctionComponent<any> = props => {
...
+              </Title>
+              <EmptyStateBody>
+                Statistics are not enabled. To activate statistics, set
```
Comment (pruivo) [inline src/app/GlobalStats/GlobalStats.tsx:119]: suggestion: `Statistic disabled. To enable statistics, ...`

File: src/app/GlobalStats/GlobalStats.tsx (hunk 1 above)
Comment (karesti) [reply src/app/GlobalStats/GlobalStats.tsx:116]: ok

File: src/app/GlobalStats/GlobalStats.tsx (hunk 2 above)
Comment (karesti) [reply src/app/GlobalStats/GlobalStats.tsx:119]: ok

**Comment by karesti** [comment]
@pruivo you can review again

**Comment by pruivo** [comment]
integrated! htanks @karesti !

**Pull request merged**